PROCESS.md

```diff
@@ -0,0 +1,171 @@
+# JSON Schema Specification Development Process
+This document formally defines the process used to evolve the JSON Schema
+specification as of the first stable release in 2023. It applies to the core
+specification and the specifications for any dialects and vocabularies
+maintained by the JSON Schema Org, but it doesn't necessarily apply to
+everything the JSON Schema Org maintains. For example, media type registration
+follows the IETF process. Certain components used within JSON Schema, such as
+Relative JSON Pointer, may also follow a separate process.
+
+This process doesn't apply to third-party dialects, vocabularies, or other
+extensions. However, third-parties are welcome to use this as a basis for the
+process they use for their specifications.
+
+## Canonical URLs
+There MUST be a stable canonical URL for referencing any specification that
+follows this process. If the spec is made available from any other URLs, they
+SHOULD redirect to the canonical URL. If the canonical URL is changed in the
+future, all previous canonical URLs MUST remain accessible as redirects to the
+current URL.
+
+## Mutable Spec
+There is only one version of the specification and it can be updated at any time
+as long as those changes don't violate the compatibility rules identified by the
+stability model. Bug fixes, clarifications, and other non-functional spec
+updates can be merged at any time even for stable features. A change-log blog
+post SHOULD be prepared quarterly to give visibility to any changes that were
+merged in the last quarter.
+
+## The Stability Model
+Every feature in the spec has a release status. It's either evolving, stable, or
+deprecated. Flags are used to show the status a feature is in. If it has a year
+flag (such as `2024`) that's the year the feature reached stability. The other
+flags indicate a feature that is not yet stable or is deprecated.
+
+If a feature doesn't have a flag, it's considered stable. A feature can only be
+unflagged if it was stable in the initial stable release in 2023. Any functional
+change to the spec after the initial stable release MUST have a flag indicating
+it's status.
```

Comment:
You quoted the answer immediately before asking the question, so I'm not sure how to answer.

> A feature can only be unflagged if it was stable in the initial stable release in 2023.

I can expand on that if it's not clear enough, but I'm not sure what's not clear. Does this discussion with @gregsdennis help? https://github.com/json-schema-org/json-schema-spec/pull/1348#discussion_r1025595948

Reply:
I see. I'll try to reword to avoid that misunderstanding. Removing or changing flags isn't disallowed, but compatibility rules apply. So, you couldn't remove a stable flag (such as `2024`) because it would be a backward compatible change, but you could change a `STAGE-2` to a `2024` because `STAGE-2` is allowed backward-incompatible changes. Similarly, you couldn't remove a `STAGE-2` flag without replacing it with a stable flag because that would break backward compatibility.

Reply:
Here's an example of the kind of thing I have in mind from the node.js docs https://nodejs.org/dist/latest-v19.x/docs/api/fs.html. They have stable, unstable, and deprecated features and flag them inline just like we're talking about here.